README.md

```diff
@@ -19,31 +111,108 @@ all keys that are presented in this sample:
 ```ini
 [existance]
 
+# installation files are cached here for repeated usage
 installer_cache = /var/cache/existance
 
 [exist-db]
 
+# the system user and usergroup that an installed / upgraded instance
+# will be run as
 user = existdb
-group = telota
+group = existdb-users
 
+# the instances' directories will be located within this directory 
 base_directory = /opt
+# the pattern must be congruent with the variable `instance_dir`'s value in the 
+# existctl script
 instance_dir_pattern = exist_{instance_name}_{instance_id}
+# this file serves as index of all instances and a few settings
 instances_settings = %(base_directory)s/exist_instances_settings.csv
 
+# the various log data is grouped within this directory
 log_directory = /var/logs/existdb
 
+# the default -XmX value for new instances
 XmX_default = 1024m
 
+# this list contains names of Jetty configuration files that are not to be 
+# used, e.g. because a modern web server can do the job for all instances
 unwanted_jetty_configs = jetty-ssl.xml,jetty-ssl-context.xml,jetty-https.xml
 ```
 
+There are still many opinionated values hardcoded in the tool respectively the
+accompanying script and configuration templates based on our concrete needs. 
+You're welcome to request extended configurability or to contribute patches in 
+this regard.
+
 
 ## Usage
 
+`existance` knows and dispatches to some subcommands that perform operations 
+and are described below.
+
+For a full reference of the available command-line parameters use 
+`existance --help` and `existance <subcommand> --help`. 
+
+The general parameters should only be used to override the values from the
+configuration file. Subcommand-specific parameters that are needed and not
+provided at the command line will be asked for.
+
 ### install
 
+In a nutshell this command:
+
+- proposes some sensible configuration values unless provided
+- downloads and caches an eXist-db installer if needed
+- updates the instances directory / settings file
+- invokes the installer after giving you some advices
+- performs further configurations as mentioned above
+- starts the newly installed instance
+
+E.g., on a host that is configured to serve the domain `exist.mydomain.web`, 
+after running
+
+    existance install --name my_project
+
+the instance's dashboard is available at https://exist.mydomain.web/my_project/
+and you're good to go.
+
 ### uninstall
 
+This is basically the opposite of the previous and you can get rid of an 
+instance as easy as you type:
+
+    existance uninstall --id <id>
+
 ### upgrade
 
```

Comment:
Some versions need backups of the data an reimport it to the modernized system. Maybe a usecase for the upgrade mechanism?

Reply:
as above?